hack/update-codecgen.sh

```diff
@@ -41,6 +41,52 @@ function cleanup {
 }
 trap cleanup EXIT
 
+# Sort all files in the dependency order.
+number=${#generated_files[@]}
+for (( i=0; i<${number}; i++ )); do
```

Comment:
nit: no need for `$` on variables in `((`. (You're inconsistent here between `i` and `number`.